link_shortener/api/api_create.py

```diff
@@ -0,0 +1,45 @@
+'''
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
+'''
+from datetime import date
+from decouple import config
+from json import loads
+
+from sanic import Blueprint
+from sanic.response import json
+
+from link_shortener.commands.create import create_link
+
+
+api_create_blueprint = Blueprint('api_create')
+
+
+@api_create_blueprint.route('/api/links', methods=['POST'])
+async def api_create_link(request):
+    try:
+        token = request.headers['Bearer']
+        if (token != config('ACCESS_TOKEN')):
```

Comment:
A decorator would be elegant, but make the API hard to unit test. I wrote a simple function to replace this block, please let me know if we can use it (In which case I'd replace it everywhere).

Reply:
Looks good to me :+1: 

Just one thing. There is a little bug that if the token is correct, it returns `None` that results in a `TypeError` in `api_retrieve_links`. You probably need to check whether the return value `is not None`.

(Although the same concern regarding the status codes/messages vs raiseing an exception in case of a wrong token applies here as well but I expect you to address/comment it in the other discussion.)